### PR TITLE
COMMERCE-10639

### PR DESCRIPTION
**Pull request by Brunno0liveira**
https://issues.liferay.com/browse/COMMERCE-10639

### CODE REVIEW

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductConfigurations.testcase [hunk 1]
```diff
@@ -1444,6 +1444,59 @@ definition {
...
+		var responseToParse = CommerceJSONProductsAPI._getCommerceFieldValue(
+			field = "uuid",
+			groupName = "Minium",
+			title = "Product Detail");
+
+		var uuid = JSONUtil.getWithJSONPath("${responseToParse}", "$..['items'][0].uuid");
+
+		var link = selenium.getHtmlNodeText("//*[contains(text(),'${uuid}')][1]");
+
+		Open.openInTheNewTab(locator1 = "${link}");
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductConfigurations.testcase:1497]:
```suggestion
		Open.openNoErrorNoSPARefresh(locator1 = "${link}");
```

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:relevant

**Comment by Brunno0liveira** [comment]
ci:test:commerce-functional

**Comment by lorenzolapescara** [comment]
ci:forward

**Pull request closed without merging**